packages/flutter_tools/lib/src/custom_devices/custom_device_config.dart

```diff
@@ -53,9 +56,8 @@ class CustomDeviceRevivalException implements Exception {
   const CustomDeviceRevivalException(this.message);
 
   const CustomDeviceRevivalException.fromDescriptions(
-    String fieldDescription,
-    String expectedValueDescription
-  ) : message = 'Expected $fieldDescription to be $expectedValueDescription.';
+      String fieldDescription, String expectedValueDescription)
```

Comment:
prefer previous formatting, per our style guide https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo

Reply:
reverted the formatting changes